packages/taquito/src/batch/rpc-batch-provider.ts

```diff
@@ -72,6 +74,14 @@ export class OperationBatch extends OperationEmitter {
     return this;
   }
 
+  withTransferTicket(params: TransferTicketParams) {
+    if (validateAddress(params.destination) !== ValidationResult.VALID) {
```

Comment:
thank you for the catch :D 
i'm not sure where that would be hit from and when i ran my tests I thought this would have been used but no error was given